explorer/src/utils/index.ts

```diff
@@ -18,7 +18,16 @@ type MixnodeRowType = {
   host: string
   layer: string
 }
+type GatewayRowType = {
+  id: string
+  owner: string
+  identity_key: string
+  bond: number
+  host: string
+  location: string
+}
 
+type GatewayRows = GatewayRowType[];
 type MixnodeRows = MixnodeRowType[];
```

Comment:
I wouldn't worry about creating extra types like this. Whenever something is expecting an array of something just specify `something[]` at the time it's needed. Just a note. Don't worry about changing this, more of a note.